packages/entando-apimanager/modules/api/apiManager.js

```diff
@@ -119,6 +119,7 @@ const getRequestParams = (request) => {
       ...contentTypeHeader,
       ...request.headers,
     },
+    credentials: (request.useCredentials == true) ? 'include' : 'omit',
```

Comment:
Beyond the use of `==` that is discouraged, the main question is: why `credentials` can never have the default value of `same-origin`? (see https://developer.mozilla.org/en-US/docs/Web/API/Request/credentials)?

If we should take into account `same-origin` as well, I think it's easier to pass the exact value of `credentials` in the `makeRequest` object parameter.

Reply:
`==` should be replaced by `===`?

The credentials attribute tells the browser to send or not the Cookie Session.
In the case for API requests we are never using it on the BE, because we always override the session user based on Access Token ir present. If not present we always set it to Guest user.

There is one exception that exceptionally we need to explicitly enable this property in order to save the Session Cookie for using when we need to retrieve protected images.

Also, we have evidenced several security issues related to the usage of the Cookie, so if we don't use it, it would be wise to explicity disable it.

In Openshift, app-builder and entando-de-app run on the same origin, hence using the default value would pose a security vulnerability.

Reply:
OK @ffleandro , replace `==` with `===` and I'll approve then

Reply:
do you think it's worth to add what you wrote here as documentation?

Reply:
Sure! What would be correct section for this information?

Reply:
apiManager's README.md (not the global one). There's the request object described there, so that would be the perfect place to add it

Reply:
Done!